src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -329,30 +329,34 @@ public Command tuneSetpoints(
 
   @Override
   public void periodic() {
+    if (!Robot.isCompetition) {
+      Logger.recordOutput(
+          "Elevator/Motor1/Position (Degrees)",
+          elevatorMotor1.getPosition().getValue().in(Degrees));
+
+      Logger.recordOutput(
+          "Elevator/Motor2/Velocity", elevatorMotor2.getVelocity().getValue().in(DegreesPerSecond));
+
+      Logger.recordOutput("Elevator/Motor2/Percent Out", elevatorMotor2.get());
+      Logger.recordOutput(
+          "Elevator/Motor2/Voltage (Volts)", elevatorMotor2.getMotorVoltage().getValue().in(Volts));
```

Comment:
Be nice to log volts just in case we have some issue with follow or something, you never know